docs/concepts/03_web3names.md

```diff
@@ -1,35 +1,35 @@
 ---
 id: web3names
-title: Web3 names
+title: web3names
 ---
 
-Web3 names are user-friendly aliases for KILT DIDs.
+In short, web3names are user-friendly aliases for KILT DIDs.
 They serve the same purpose that domain names do for IP addresses: who knows the IP address under the `kilt.io` domain name? 🤷🏽‍♀️
-There is a one-to-one relationship between DIDs and Web3 names.
-This means that a KILT DID can be linked to at most one Web3 name, and a Web3 name can be claimed only by one DID.
+There is a one-to-one relationship between DIDs and web3names.
+This means that a KILT DID can be linked to at most one web3name, and a web3name can be claimed only by one DID.
 
-Each Web3 name is globally unique within the KILT space, and is composed of a sequence of a minimum of 3 to a maximum of 32 characters taken from a specific character set to enhance human redability and reduce the chances of *two Web3 names looking the same, despite being different*.
+Each web3name is globally unique within the KILT space, and is composed of a sequence of a minimum of 3 to a maximum of 32 characters taken from a specific character set to enhance human redability and reduce the chances of *two web3names looking the same, despite being different*.
 The character set include only:
 - *lowercase letters*, from `a` to `z`
 - *digits* from `0` to `9`
 - the symbols `-` and `_`
 
-A regex that would match all and only the allowed Web3 names would be the following:
+A regex that would match all and only the allowed web3names would be the following:
 
 ```
 ^[a-z0-9\_\-]{3,32}$
 ```
 
-In the global URI space, Web3 names are prefixed with the `w3n:` URI namespace.
-For example, the full URI for the Web3 name `example-web3name` is `w3n:example-web3name`.
+In the global URI space, web3names are prefixed with the `w3n:` URI namespace.
+For example, the full URI for the web3name `example-web3name` is `w3n:example-web3name`.
 
-### Linking accounts to a Web3 name
+### Linking Multiple Accounts to a web3name
```

Comment:
isnt it linking accounts to a DID? Its only transitively linked to the web3name through the DID

Reply:
Yes, but the goal of this section is to explain how to to show names for accounts.